### PR TITLE
Update NATS BBE content

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$subject

### CODE REVIEW

File: examples/nats-basic-reply/nats-basic-reply.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # NATS service - Send reply to request message
 
-In this example, the NATS service is used to send replies to incoming request messages. In order to execute this example, it is required that a NATS server is up and running on its default host, port, and cluster. For instructions on installing the NATS server, go to [NATS Server Installation](https://docs.nats.io/nats-server/installation). 
+NATS supports the Request-Reply pattern using its core message distribution model, publish and subscribe. A request is sent to a given subject and consumers listening to that subject can send responses to the reply subject. In this example, the NATS service is used to send replies to incoming request messages.
```
Comment (praneesha) [inline examples/nats-basic-reply/nats-basic-reply.md:3]:
```suggestion
NATS supports the Request-Reply pattern using its core message distribution model, publish, and subscribe. A request is sent to a given subject and consumers listening to that subject can send responses to the reply subject. In this example, the NATS service is used to send replies to incoming request messages. 
```

File: examples/nats-basic-request/nats-basic-request.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # NATS client - Send request message
 
-In this example, the NATS client is used to send a request to a subject. In order to execute this example, it is required that a NATS server is up and running on its default host, port, and cluster. For instructions on installing the NATS server, go to [NATS Server Installation](https://docs.nats.io/nats-server/installation). 
+NATS supports the Request-Reply pattern using its core message distribution model, publish and subscribe. A request is sent to a given subject and consumers listening to that subject can send responses to the reply subject. In this example, the NATS client is used to send a request to a subject.
```
Comment (praneesha) [inline examples/nats-basic-request/nats-basic-request.md:3]:
```suggestion
NATS supports the Request-Reply pattern using its core message distribution model, publish, and subscribe. A request is sent to a given subject and consumers listening to that subject can send responses to the reply subject. In this example, the NATS client is used to send a request to a subject.
```

File: examples/nats-client-basic-auth/nats-client-basic-auth.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
 # NATS client - Basic authentication
 
-In this example, the underlying connection of the publisher is secured with Basic Auth. 
+NATS client connections can be authenticated in many ways. One of them is using username and password credentials. In this example, the underlying connection of the publisher is secured with basic authentication.
```
Comment (praneesha) [inline examples/nats-client-basic-auth/nats-client-basic-auth.md:3]:
```suggestion
NATS client connections can be authenticated in many ways. One of them is by using the username and password credentials. In this example, the underlying connection of the publisher is secured with Basic Authentication. 
```

File: examples/nats-client-secure-connection/nats-client-secure-connection.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
 # NATS client - SSL/TLS
 
+The NATS client connections can be secured by encrypting with TLS. In this example, the underlying connection of the listener is secured with basic authentication.
```
Comment (praneesha) [inline examples/nats-client-secure-connection/nats-client-secure-connection.md:3]:
```suggestion
The NATS client connections can be secured by encrypting with TLS. In this example, the underlying connection of the listener is secured with Basic Authentication.
```

File: examples/nats-service-basic-auth/nats-service-basic-auth.md [hunk 5]
```diff
@@ -1,9 +1,11 @@
 # NATS service - Basic authentication
 
-In this example, the underlying connections of the subscriber is secured with Basic Auth.
+NATS client connections can be authenticated in many ways. One of them is using username and password credentials. In this example, the underlying connection of the listener is secured with basic authentication.
```
Comment (praneesha) [inline examples/nats-service-basic-auth/nats-service-basic-auth.md:3]:
```suggestion
NATS client connections can be authenticated in many ways. One of them is by using the username and password credentials. In this example, the underlying connection of the listener is secured with Basic Authentication.
```

File: examples/nats-client-secure-connection/nats-client-secure-connection.md (hunk 4 above)
Comment (aashikam) [reply examples/nats-client-secure-connection/nats-client-secure-connection.md:3]: going with existing one,

**Review by praneesha — APPROVED** [review]

**Pull request merged**